preprocessors/content-categoriser.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/content-categoriser.schema.json",
+    "type": "object",
+    "title": "Content Categoriser",
+    "description": "Classifier for categorising the content into grapic, text, chart or other.",
```

Comment:
What's the reverse domain name this will be under? `ca.mcgill.a11y.image.preprocessor.contentCategoriser`?

Reply:
```suggestion
    "description": "Classifier for categorising the content into graphic, text, chart or other.",
```

Reply:
It is not clear to me what reverse domain means.

Reply:
https://en.wikipedia.org/wiki/Reverse_domain_name_notation
(for example, the reverse domain in Juliette's comment above)

Reply:
Ok, but I still don't understand the question or the issue with current name.